src/commands/utility/info/category.js

```diff
@@ -0,0 +1,36 @@
+const { Command, SwitchbladeEmbed, CommandError } = require('../../../')
+const moment = require('moment')
+
+module.exports = class InfoCategory extends Command {
+  constructor (client) {
+    super({
+      name: 'category',
+      parent: 'info',
+      requirements: { guildOnly: true },
+      parameters: [{
+        type: 'channel',
+        acceptCategory: true,
```

Comment:
It doesn't accept category IDs, like 445631099440463872 for example

Reply:
Yeah, because the channel parameter never did.

Reply:
Oh